### DISCUSSION TITLE
Ping One for Customers support

### DISCUSSION THREAD

**Discussion by rcastag** [Web3Auth/Web3Auth#538] [Help]
We use Ping One for Customers for OAuth2/ OIDC. How could we go about integrating with Web3Auth.

**Comment by shahbaz17** [comment]
Hey @rcastag 

Please see https://github.com/orgs/Web3Auth/discussions/476#discussioncomment-3471701

Also, it would be similar to https://web3auth.io/docs/guides/firebase guide, simply replace the Firebase part with Ping One to get the `idToken` and pass it in the same manner.

**Reply by rcastag** [reply]
Thanks!
